doc/nrf/libraries/networking/azure_iot_hub.rst

```diff
@@ -99,8 +99,7 @@ To connect to Azure IoT Hub without using DPS, complete the following minimum re
    If the ``device_id.size`` buffer size is zero, the compile-time option :kconfig:option:`CONFIG_AZURE_IOT_HUB_DEVICE_ID`` is used.
 #. Make sure that the device is already registered with your Azure IoT Hub, or follow the instructions in `Registering the device with Azure IoT Hub`_.
 #. Set the :kconfig:option:`CONFIG_MQTT_HELPER_SEC_TAG` option to the security tag used in :ref:`azure_iot_hub_flash_certs`.
```

Comment:
Seems to be missed from update, and why are these being updates anyway? Commit title says a username buffer Kconfig is being updated, this has no relation

Reply:
This is the primary sec_tag, comes with no _SECONDARY_

Reply:
This needs splitting up into 2 commits, the changes here have no relation to a username buffer size